scripts/ci/libraries/_initialization.sh

```diff
@@ -134,8 +134,8 @@ function initialization::initialize_base_variables() {
     # otherwise it will use files/airflow-breeze-config/init.sh
     export INIT_SCRIPT_FILE=${INIT_SCRIPT_FILE:=""}
 
-    # Read airflow version from the version.py
-    AIRFLOW_VERSION=$(grep version "${AIRFLOW_SOURCES}/airflow/version.py" | awk '{print $3}' | sed "s/['+]//g")
+    # Read airflow version from the setup.py.
+    AIRFLOW_VERSION=$(python "${AIRFLOW_SOURCES}/setup.py" --version)
```

Comment:
I'd prefer not to run Python here if possible. While this is a reasonable expectation that Python is installed in the hos, there are still some cases where people will have python2 by default for example and i would rather keep it bash only. There is another case in the future where we will use those scripts in code spaces environment and vs code to run docker-only environment and i would not like to have to depend on Python there.

Reply:
Sure.

Reply:
> # scripts/ci/libraries/_initialization.sh: line 138: python: command not found

Failed anyway :)

Reply:
Of course that all untill we rewrite breeze in python - which might change the whole approach: #12282